test/test_automated_install.py

```diff
@@ -334,61 +409,67 @@ def test_FTL_binary_installed_and_responsive_no_errors(Pihole):
+    mock_command_2('ip', {'-6 address': ('inet6 fe80::d210:52fa:fe00:7ad7/64 scope link', '0')}, Pihole)
     detectPlatform = Pihole.run('''
     source /opt/pihole/basic-install.sh
     useIPv6dialog
     ''')
     expected_stdout = 'Unable to find IPv6 ULA/GUA address, IPv6 adblocking will not be enabled'
     assert expected_stdout in detectPlatform.stdout
 
+
 def test_IPv6_only_ULA(Pihole):
     ''' confirms IPv6 blocking is enabled for ULA addresses '''
     # mock ip -6 address to return ULA address
-    mock_command_2('ip', {'-6 address':('inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
+    mock_command_2('ip', {'-6 address': ('inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
     detectPlatform = Pihole.run('''
     source /opt/pihole/basic-install.sh
     useIPv6dialog
     ''')
     expected_stdout = 'Found IPv6 ULA address, using it for blocking IPv6 ads'
     assert expected_stdout in detectPlatform.stdout
 
+
 def test_IPv6_only_GUA(Pihole):
     ''' confirms IPv6 blocking is enabled for GUA addresses '''
     # mock ip -6 address to return GUA address
-    mock_command_2('ip', {'-6 address':('inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
+    mock_command_2('ip', {'-6 address': ('inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
     detectPlatform = Pihole.run('''
     source /opt/pihole/basic-install.sh
     useIPv6dialog
     ''')
     expected_stdout = 'Found IPv6 GUA address, using it for blocking IPv6 ads'
     assert expected_stdout in detectPlatform.stdout
 
+
 def test_IPv6_GUA_ULA_test(Pihole):
     ''' confirms IPv6 blocking is enabled for GUA and ULA addresses '''
     # mock ip -6 address to return GUA and ULA addresses
-    mock_command_2('ip', {'-6 address':('inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global\ninet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
+    mock_command_2('ip', {'-6 address': ('inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global\ninet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
     detectPlatform = Pihole.run('''
     source /opt/pihole/basic-install.sh
     useIPv6dialog
     ''')
     expected_stdout = 'Found IPv6 ULA address, using it for blocking IPv6 ads'
     assert expected_stdout in detectPlatform.stdout
 
+
 def test_IPv6_ULA_GUA_test(Pihole):
     ''' confirms IPv6 blocking is enabled for GUA and ULA addresses '''
     # mock ip -6 address to return ULA and GUA addresses
-    mock_command_2('ip', {'-6 address':('inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global\ninet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
+    mock_command_2('ip', {'-6 address': ('inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global\ninet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
```

Comment:
The extra long ones like this could easily be split on the newline cleanly

```
mock_command_2('ip', {'-6 address': ('inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global\n'
   'inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global', '0')}, Pihole)
```
is valid I think

Reply:
This is more readable:
```python
mock_command_2(
    'ip',
    {
        '-6 address': (
            'inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global\n'
            'inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global',
            '0'
        )
    },
    Pihole
)
```

Reply:
alt style that adheres to 79 chars:
```
    cmd_args = {'-6 address':
                ('inet6 2003:12:1e43:301:d210:52fa:fe00:7ad7/64 scope global\n'
                 'inet6 fda2:2001:5555:0:d210:52fa:fe00:7ad7/64 scope global',
                '0')
                }
    mock_command_2('ip', cmd_args, Pihole)
```